common/app/model/dotcomrendering/DotcomRenderingDataModel.scala

```diff
@@ -391,9 +394,12 @@ object DotcomRenderingDataModel {
       // behaviour. At the moment we are seeing interactive articles with other
       // design types due to CAPI format logic. But interactive design should
       // always take precendent (or so we think).
-      content.metadata.contentType match {
-        case Some(DotcomContentType.Interactive) => originalFormat.copy(design = InteractiveDesign)
-        case _                                   => originalFormat
+      if (content.metadata.contentType.contains(DotcomContentType.Interactive)) {
+        originalFormat.copy(design = InteractiveDesign)
+      } else if (forceLive) {
+        originalFormat.copy(design = LiveBlogDesign)
```

Comment:
I am unfamiliar with the code, and also there's no test to explain what this logic is for - what's it doing?

Reply:
I know, I did think tests, but I got lazy when there were none already there. I'll add something, there's quite a bit of logic in here now and perhaps getting out of hand 👍